src/helpers/rangy.ts

```diff
@@ -16,7 +16,7 @@ export const findTextInRange = (
   text: string,
   range: RangyRange = rangy.createRange()
 ): RangyRange[] => {
-  const foundMatch = range.findText(text, {
+  const foundMatch = range.findText(text.trim(), {
```

Comment:
Reason for this is mainly [this line](https://github.com/timdown/rangy/blob/1e55169d2e4d1d9458c2a87119addf47a8265276/src/modules/rangy-textrange.js#L885)
Because of it every text that has a leading space and is right after math is never found. 
Some tags inside mathml fit [this requirement](https://github.com/timdown/rangy/blob/1e55169d2e4d1d9458c2a87119addf47a8265276/src/modules/rangy-textrange.js#L675) which injects a new line to the string that rangy is using to compare with the query.